Aptoide/aptoide-client-v8
# [ASV-1655] - Reaction bar dismiss on scroll.

**What does this PR do?**

   This PR aims to fix a bug regarding scrolling on the editorial list. This was achieved by blocking the scrolling when long pressing the react button, and unlocking the scroll once the reaction popup has been dismissed.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] EditorialListFragment.java

**How should this be manually tested?**

  Long press on the react button inside the Editorial List and without letting go try to scroll. The list should be scroll-blocked. Try dismissing the popup, and try to scroll. The list should be scroll-free.

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-1655



**Code Review Checklist**
